surf/src/vcs/git/repo.rs

```diff
@@ -107,8 +112,13 @@ impl<'a> RepositoryRef<'a> {
     /// # Errors
     ///
     /// * [`Error::Git`]
-    pub fn list_tags(&self) -> Result<Vec<Tag>, Error> {
-        RefGlob::Tag
+    pub fn list_tags(&self, scope: RefScope) -> Result<Vec<Tag>, Error> {
+        let ref_glob = match scope {
```

Comment:
Can we move these two `match`es into functions on `RefGlob`? i.e. `RefGlob::tag(scope)` and `RefGlob::branch(scope)`

Reply:
Fixed in https://github.com/radicle-dev/radicle-surf/pull/196/commits/639b54cd3cc3abd1bafec8381edc45db7d569d87